### DISCUSSION TITLE
`播放器投影` 功能支持迷你播放器

### DISCUSSION THREAD

**Discussion by Geeyun-JY3** [the1812/Bilibili-Evolved#1379] [功能建议 / Ideas]
偶然发现迷你播放器官方效果有个阴影（夜间模式下特别明显）。要不让这个投影也变成主题色投影做进 `播放器投影` 功能里去？
![image](https://user-images.githubusercontent.com/49396421/103170154-1b3c9880-487d-11eb-8719-7a82aa4a348d.png)

**Comment by the1812** [comment]
主题色投影会适配过去, 但是这里夜间下应该是暗色的, 属于未适配了 (旧版的 mini 播放器是适配过的

**Reply by LonelySteve** [reply]
哎，不知道有多少样式代码已经失效了（

**Reply by the1812** [reply]
mini那应该就一个投影 我已经修了

**Reply by LonelySteve** [reply]
嗯，我只是想吐槽一下，现在夜间模式的样式越堆越多，一千行左右的代码都堆了 13 个文件，就算是压缩过的 dark.min.css 也足足有 500KB 左右，旧有的样式代码如果彻底失效基本上无从得知，到时候必须彻底重写一遍，否则无用的代码会越积越多，项目体积也会越来越大，失去控制，出现类似 #1497 这样的麻烦。

而且虽说弄了离线版/在线版，但是我发现这些版本的体积都差不多，夜间样式代码也都包含在其中，我觉得对于在线版，这是不必要的，只需要在用户首次启用夜间模式的时候把样式代码缓存到本地就行了。

**Reply by the1812** [reply]
> 到时候必须彻底重写一遍

有生之年(

> 但是我发现这些版本的体积都差不多

曾经是有明显区别的, 因为在线版不包含那些功能模块的文件, 开启的时候才下载
不过那时候没有 jsDelivr, 即使下载一个 50KB 的代码也要等很久 (后来直接就墙了)
2.0会不再做这个区分, 利用 jsDelivr 来做缓存就行了

**Comment by Geeyun-JY3** [comment]
新版播放页的迷你播放器似乎已无投影。

**Reply by Geeyun-JY3** [reply]
顺嘴一提，现在旧版播放页的迷你播放器也已无此投影。
